core/common/src/main/java/alluxio/resource/CloseableResource.java

```diff
@@ -11,13 +11,15 @@
 
 package alluxio.resource;
 
+import java.io.Closeable;
+
 /**
  * A {@code CloseableResource<T>} is a wrapper around a resource of type {@code T} which must do
  * some sort of cleanup when it is no longer in use.
  *
  * @param <T> the type of the wrapped resource
  */
-public abstract class CloseableResource<T> implements AutoCloseable {
+public abstract class CloseableResource<T> implements Closeable {
```

Comment:
Why is this change?

Reply:
`Closeable` can be registered to `Closer`